### ISSUE TITLE
Up-Arrow crashes PowerShell (without PSReadLine)

### ISSUE THREAD

**Issue by Jaykul** [PowerShell/PowerShell#1586]
In PowerShell in Bash on Ubuntu on Windows ...
## Steps to reproduce
1. Up Arrow -- produces "A"
2. Remove-Module PSReadLine
3. Up Arrow
4. Up Arrow # will produce "Aborted (core dumped)"
## Expected behavior

Up Arrow should navigate through history.
## Actual behavior

Up Arrow types "A" or crashes PowerShell.
## Environment data

``` powershell
> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.10032.0
PSEdition                      PowerShellCore
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   3.0.0.0
GitCommitId                    v6.0.0-alpha.7
CLRVersion
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

```

**Comment by andschwa** [comment]
@Jaykul For WSL issues, especially for .NET Core on WSL, we should follow up with [BashOnWindows](https://github.com/Microsoft/BashOnWindows).

**Comment by rhino-corey** [comment]
Was the issue with arrow keys typed into bash ever resolved? It's happening for me in Powershell 5.1 when shell'd into a docker container. Is there a continuation of this issue somewhere else?
